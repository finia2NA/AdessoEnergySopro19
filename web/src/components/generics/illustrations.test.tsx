import React from 'react';

test('Dully', () => {
  expect(1 + 1).toBe(2);
});
