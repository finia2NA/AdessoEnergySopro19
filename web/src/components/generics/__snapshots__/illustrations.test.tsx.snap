// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Illustration of type Auth matches with snapshot 1`] = `
<svg
  class="illustration"
  data-testid="Auth"
>
  auth.svg
</svg>
`;

exports[`Illustration of type NoSelected matches with snapshot 1`] = `
<svg
  class="illustration"
  data-testid="NoSelected"
>
  paper.svg
</svg>
`;
